applications: serial_lte_modem: xsleep doc note (#4822)
nrfconnect/sdk-nrf: applications/serial_lte_modem/doc/Generic_AT_commands.rst

The ``#XSLEEP`` command makes the nRF91 development kit go into idle or sleep mode.

> If you are going to do power measurements on the nRF9160 DK while running the SLM application it is recommended to disable peripherals that are not in use.

Reviewer: ```suggestion
If you are going to do power measurements on the nRF9160 DK while running the SLM application, it is recommended to disable unused peripherals.
```

> By default UART2 is enabled in the ``nrf9160dk_nrf9160ns.overlay`` so disable the UART2 by switching the status.

Reviewer: ```suggestion
By default, UART2 is enabled in the :file:`nrf9160dk_nrf9160ns.overlay` file so disable the UART2 by switching the status.
```

> Change the line ``status = "okey"`` to ``status = "disabled"`` then save the ``nrf9160dk_nrf9160ns.overlay`` file to make sure you will get the expected power consumption numbers when doing the measurements.

Reviewer: ```suggestion
Change the line ``status = "okay"`` to ``status = "disabled"`` and save the :file:`nrf9160dk_nrf9160ns.overlay` file to make sure you get the expected power consumption numbers when doing the measurements.
```
